src/components/Jobs/JobList/JobList.tsx

```diff
@@ -40,6 +40,19 @@ const JobList = (props: Props) => {
     );
   }
 
+  if (jobSearchList.length === 0 && jobPaginationList.length === 0) {
+    return (
+      <div className="container my-5">
+        <div className="row text-center">
+          <p>Ops..! The job was not found !!</p>
+          <p>
+            Back to home: <a href="/">Home</a>
+          </p>
+        </div>
+      </div>
+    );
+  }
+
   if (jobSearchList.length > 0) {
```

Comment:
Do we need conditions here?
Same code below

Reply:
Fixed